course_content/notebooks/numpy_intro.ipynb

```diff
@@ -445,8 +447,8 @@
      "metadata": {},
      "source": [
       "#### The Array Object: Summary of key  points\n",
-      " * properties : shape, dtype\n",
-      " * creation : array(list), ones, zeros, arange, linspace\n",
+      " * properties : `shape`, `dtype`\n",
+      " * creation : `array([list])`, `ones`, `zeros`, `arange`, `linspace`\n",
```

Comment:
"array(list)" confused me when I first read it, so I had a go at making it clearer. Not convinced I've managed though...